src/pages/components/TabItem.qml

```diff
@@ -36,13 +32,9 @@ BackgroundItem {
 
```

Comment:
Remove the clip couple of lines above and adjust thumbnailSize of the DeclarativeWebPage to match to the TabItem's size. There is currently unnecessary cost coming from this.

Namely:
```
        width: Screen.width - (largeScreen ? (2 * Theme.horizontalPageMargin) : 0)
        height: largeScreen ? Theme.itemSizeExtraLarge + (2 * Theme.paddingLarge) : Screen.height / 5
```